install_config/topics/configuring_gce_objects-proc.adoc

```diff
@@ -0,0 +1,171 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Configuring Google Compute Engine objects
+
+Integrating {product-title} with Google Compute Engine (GCE) requires the
+following {provider} components or services to create a highly-available and
+full featured environment.
+
+GCE project::
+
+A GCE project is the base level organizing entity and forms the basis for
+creating, enabling and using all GCE services. This includes managing APIs,
+enabling billing, adding and removing collaborators, and managing permissions.
++
+For more information, see  link:https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy#projects[The project resource in the GCE documentation].
++
+[IMPORTANT]
+====
+To ensure a continuous successful integration, the project ID must be the same for all components and services. 
+====
+
+Billing::
+
+Billing ensures new resources can be created. The new project can be linked to
+an existing project or financial information can be entered.
+
+Cloud identity and access management::
+
+Access to the various infrastructure, service resources, and
+fine-grained roles are available using the IAM service.
++
+You can create service accounts with specific permissions, then use them to
+deploy infrastructure components instead of regular users. Also, you can create
+roles to limit access to different users or service accounts.
++
+GCE instances use service accounts to allow applications to call GCE APIs. For
+example, {product-title} node hosts can call the GCE disk API to provide a
+persistent volume to an application.
++
+Deploying {product-title} requires the proper permissions. A user must be able
+to create service accounts, cloud storage, instances, images, templates, Cloud
+DNS entries, and deploy load balancers and health checks. Delete permissions are
+also helpful in order to be able to redeploy the environment while testing.
++
+For more information, see link:https://cloud.google.com/compute/docs/access/[the Access cloud overview section of the GCE documentation].
+
+SSH keys::
+
+GCE injects SSH public keys as authorized keys to be able to log in using SSH in
+the created instances. The list of SSH keys can be configured per instance or
+per project.
+
+Configuring GCE regions and zones::
+
+GCE has a global infrastructure that covers regions and availability
+zones.
```

Comment:
This doesn't actually describe what is needed for regions and zones. Is there alternative info to put here that explains why this is in the prerequisites?